bazel/toolchains/multirun_deps.bzl

```diff
@@ -0,0 +1,10 @@
+"""multirun_deps"""
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+def multirun_deps():
+    git_repository(
+        name = "com_github_ash2k_bazel_tools",
+        commit = "4e045b9b4e3e613970ab68941b556a356239d433",
+        remote = "https://github.com/ash2k/bazel-tools.git",
+    )
```

Comment:
nit: using a git_repository dep is harder for us to add to our own cache. (Implementing automatically adding `http_archive` rules to the edgeless archive is still an open todo, so this will only become an issue then.) I think using `http_archive` for everything makes this a bit easier.
Also it is less obvious how to ensure it is actually reproducible.
The current code leads to the message:
```
DEBUG: Rule 'com_github_ash2k_bazel_tools' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1672749937 +1100"
DEBUG: Repository com_github_ash2k_bazel_tools instantiated at:
  /home/malte/projects/clones/constellation/WORKSPACE.bazel:143:14: in <toplevel>
  /home/malte/projects/clones/constellation/bazel/toolchains/multirun_deps.bzl:6:19: in multirun_deps
Repository rule git_repository defined at:
  /home/malte/.cache/bazel/_bazel_malte/e58bd5cf140e0accfbab91a2f501a0a3/external/bazel_tools/tools/build_defs/repo/git.bzl:176:33: in <toplevel>
```

You can create a `.tar.gz` from any GitHub commit using this scheme:
https://github.com/ash2k/bazel-tools/archive/4e045b9b4e3e613970ab68941b556a356239d433.tar.gz